whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -417,19 +405,20 @@ private IpEntry lookupParentIpEntry(final IpInterval ipInterval) {
         if (ipInterval instanceof Ipv4Resource) {
             final List<Ipv4Entry> firstLessSpecific = ipv4Tree.findFirstLessSpecific((Ipv4Resource) ipInterval);
             if (firstLessSpecific.isEmpty()) {
-                throw new IllegalStateException("No parent for " + ipInterval.toString());
+                throw new IllegalStateException("No parent for " + ipInterval);
             }
             return firstLessSpecific.get(0);
-        }
-
-        if (ipInterval instanceof Ipv6Resource) {
-            final List<Ipv6Entry> firstLessSpecific = ipv6Tree.findFirstLessSpecific((Ipv6Resource) ipInterval);
-            if (firstLessSpecific.isEmpty()) {
-                throw new IllegalStateException("No parent for " + ipInterval.toString());
+        } else {
```

Comment:
here maybe it is more clear "else if()" than "else { if ()" ??

Reply:
I didn't want a dangling else at the bottom, so added all braces to be really clear.